src/api/java/com/minecolonies/api/util/constant/TranslationConstants.java

```diff
@@ -412,21 +412,25 @@
     @NonNls
     public static final String OUT_OF_COLONY                                                        = "com.minecolonies.coremod.workorder.outofcolony";
     @NonNls
-    public static final String NO                                                                   = "com.minecolonies.coremod.entity.citizen.no.";
+    public static final String NO                                              = "com.minecolonies.coremod.entity.citizen.no.";
```

Comment:
..Why did you change the whitespace?

Reply:
thats autoformatting of our defined formatting style xD

Reply:
but it's not changing the rest of the file?